### PR TITLE
llama2 readme update performance

### PR DESCRIPTION
**Pull request by ZhongYingMatrix**
### What changes were proposed in this pull request?

Please describe the changes you have made or proposed in this pull request.

### Why are the changes needed?

Explain the purpose or motivation behind these changes. What problem are you trying to solve?

### Does this PR introduce any user-facing change?

Specify whether this pull request introduces any changes that users will directly interact with or notice.

### How was this patch tested?

Detail the testing process you have undertaken to ensure the changes in this pull request are valid and working as intended.

### CODE REVIEW

File: atorch/examples/llama2/README.md [hunk 1]
```diff
@@ -402,4 +402,25 @@ pip install -r requirements.txt
...
 sh ds_3d_llama2_entry.sh
-```
\ No newline at end of file
+```
+
+### Performance
+- Model: Llama2-7b
+- Cluster: 8 A100-80GB GPUs in each node, with NVLink and RDMA
+
+| Method | Use LoRA | Num of GPUs | TP | PP | DP | Batch Per GPU | Grad Accu Steps | Global Batch Size | TFLOPs | Mem Max allocated(MB) |
```
Comment (skydoorkai) [inline atorch/examples/llama2/README.md:411]: Add HFU column.

**Review by workingloong — APPROVED** [review]
LGTM

File: atorch/examples/llama2/README.md [hunk 2]
```diff
@@ -402,4 +402,25 @@ pip install -r requirements.txt
 # e.x. in a 8-gpu system, to run mp-2 dp-2 pp-2, 
 # use `PIPELINE_PARALLEL_SIZE=2 MODEL_PARALLEL_SIZE=2 sh ds_3d_llama2_entry.sh`
 sh ds_3d_llama2_entry.sh
-```
\ No newline at end of file
+```
+
+### Performance
+- Model: Llama2-7b
+- Cluster: 8 A100-80GB GPUs in each node, with NVLink and RDMA
```
Comment (skydoorkai) [inline atorch/examples/llama2/README.md:409]: 400 Gb/s RDMA? or 800 Gb/s?

File: atorch/examples/llama2/README.md (hunk 1 above)
Comment (skydoorkai) [inline atorch/examples/llama2/README.md:411]: lora flops computation seems not correct.
Also for lora training, add "wrap_trainable_outmost": True, in fsdp config would increase perf. In my 8-A100 test, the printed flops increase from 196 to 239.

**Review by skydoorkai — APPROVED** [review]
lgtm

**Pull request merged**